### ISSUE TITLE
Add Mangio-crepe to RVC for training and inference.

### ISSUE THREAD

**Issue by litsa-the-dancer** [fumiama/Retrieval-based-Voice-Conversion-WebUI#67]
It would be a good idea to add mangio-crepe back into RVC. It's a good pitch extraction method and a lot of people actually like using it. Additionally it can sometimes have better sibilance and breathing, alongside with better mouth click handling which can add more naturalness. I dont think it would be that hard to add so yeah, please do. ^w^

**Comment by fumiama** [comment]
OK. Do you mean this repo?

https://github.com/Anthonyxd22/Mangio-Crepe-m1-m2

**Comment by litsa-the-dancer** [comment]
Yeah but only get the pitch extraction method mangio crepe. Mangio crepe is basically just crepe but with an adjustable hop length. Better crepe basically.

**Comment by fumiama** [comment]
OK, marked.

**Comment by TheTrustedComputer** [comment]
Just a fair warning: the current Mangio CREPE GPU implementation is broken on AMD cards, at least on my 5500 XT in a Linux environment. It completely crashes my GPU and eventually causes a hard reset of my PC. The CPU version, albeit significantly slower, should work fine.

**Comment by fumiama** [comment]
That's pretty strange.

**Comment by TheTrustedComputer** [comment]
Agreed. If anyone has the expertise (I don't) to optimize Mangio CREPE for AMD GPUs without sacrificing performance on NVIDIA, please do so. Your contributions will be greatly appreciated.

**Comment by TheTrustedComputer** [comment]
You may or may not be aware of this, but there's a fork of RVC called [Applio](https://github.com/IAHispano/Applio) that has a working implementation of Mangio CREPE on AMD GPUs. They simplified it to just CREPE, which no longer crashes my machine.

**Comment by litsa-the-dancer** [comment]
Yeah i made the suggestion to the staff. I'm well aware. I simply prefer non forked RVC since im used to it.
